scss/mixins/_pagination.scss

```diff
@@ -11,17 +11,68 @@
 
 // scss-docs-start pagination-max-items-mixin
 // Boosted mod
-// Ensure we only display `n` items:
-// - the first `n/2` items
-// - the last `n/2` items
-// - including the .active one (so maybe not the `n/2`th last item)
+// The pagination list ((m) items) should display exactly (n) items and is built in 3 parts:
+// - the begining of the list is made of the first (n/2) items
+// - the end of the list is made of the last (n/2) items (or (n/2 - 1) depending on the active element)
+// - the last part of the list is made of the active element
+// - n should be greater than 7 since we want at least two arrows, the first, the active and the last and might have 2 three dots cases for a11y and design reasons
 @mixin pagination-max-items($pagination-max-items) {
-  display: none;
+  @if $enable-responsive-pagination {
+    $pagination-max-items: max(7, $pagination-max-items);
```

Comment:
Not sure to understand why we need a max? Is it link to a specific request?

Reply:
Maximal number of items inside the minimal use case: 
- 2 arrows for next and previous
- 2 extreme numbers (the first and the last)
- 2 `...` to ensure comprehension
- 1 active item

To have exactly the same number of item everytime, we need at least 7 items inside.

Reply:
It seems that this minimal use case is not the 'real' minimum since you can have less items when there are only 2 or 3 pages to paginate for instance.

Reply:
Yes exactly, it represents the maximum number of displayed items, but this maximum can't be under 7 otherwise we would have some unwanted behavior.